File: 3053.c
#include<stdio.h>

int main(){
    int inputNum;
    scanf("%d", &inputNum);
    printf("%6lf\n%6lf", inputNum * inputNum * 3.1415926535897932384626, inputNum * inputNum * 2.0);
    return 0;
}

File: 3053.java
import java.io.*;
import java.lang.Math;

public class Main{
    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int inputNum = Integer.parseInt(br.readLine());
        System.out.printf("%6f\n%6f", (inputNum * inputNum * Math.PI), (inputNum * inputNum * 2.0));
    }
}


